version/version.go

```diff
@@ -0,0 +1,23 @@
+package version
+
+// This variable is set at build time using -ldflags parameters. For example, we typically set this flag in circle.yml
+// to the latest Git tag when building our Go apps:
+//
+// build-go-binaries --app-name my-app --dest-path bin --ld-flags "-X github.com/gruntwork-io/gruntwork-cli/version.gitTag=$CIRCLE_TAG"
+// go build -o my-app -ldflags "-X github.com/gruntwork-io/gruntwork-cli/version.getTag=v1.0.0"
+//
```

Comment:
I guess it more flexible to have a constant path for replacing instead of keep in main.go, especially for DevOps